proposal/color-4-new-spaces.md

```diff
@@ -2327,31 +2345,229 @@ plain CSS function named `"hsl"` that function is named `"hsla"` instead.
+  * If `$color` is not a [legacy color], throw an error.
 
-[legacy color]: #legacy-color
+  * Return the result of calling `color.channel($color, 'red')`.
+
+* ```
+  red($color)
+  ```
+
+  * If `$color` is not a [legacy color], throw an error.
+
+  * Return the result of calling `color.channel($color, 'red')`.
+
+### `color.green()`, `green()`
+
+* ```
+  color.green($color)
+  ```
+
+  * If `$color` is not a [legacy color], throw an error.
+
+  * Return the result of calling `color.channel($color, 'green')`.
+
+* ```
+  green($color)
+  ```
+
+  * If `$color` is not a [legacy color], throw an error.
+
+  * Return the result of calling `color.channel($color, 'green')`.
+
+### `color.blue()`, `blue()`
+
+* ```
+  color.blue($color)
+  ```
+
+  * If `$color` is not a [legacy color], throw an error.
+
+  * Return the result of calling `color.channel($color, 'blue')`.
+
+* ```
+  blue($color)
+  ```
+
+  * If `$color` is not a [legacy color], throw an error.
+
+  * Return the result of calling `color.channel($color, 'blue')`.
+
+### `color.hue()`, `hue()`
+
+* ```
+  color.hue($color)
+  ```
+
+  * If `$color` is not a [legacy color], throw an error.
+
+  * Return the result of calling `color.channel($color, 'hue')`.
+
+* ```
+  hue($color)
+  ```
+
+  * If `$color` is not a [legacy color], throw an error.
+
+  * Return the result of calling `color.channel($color, 'hue')`.
+
+### `color.saturation()`, `saturation()`
+
+* ```
+  color.saturation($color)
+  ```
+
+  * If `$color` is not a [legacy color], throw an error.
+
+  * Return the result of calling `color.channel($color, 'saturation')`.
+
+* ```
+  saturation($color)
+  ```
+
+  * If `$color` is not a [legacy color], throw an error.
+
+  * Return the result of calling `color.channel($color, 'saturation')`.
+
+### `color.lightness()`, `lightness()`
+
+* ```
+  color.lightness($color)
+  ```
+
+  * If `$color` is not a [legacy color], throw an error.
+
+  * Return the result of calling `color.channel($color, 'lightness')`.
+
+* ```
+  lightness($color)
+  ```
+
+  * If `$color` is not a [legacy color], throw an error.
+
+  * Return the result of calling `color.channel($color, 'lightness')`.
+  
+  ### `color.whiteness()`
+
+* ```
+  color.whiteness($color)
+  ```
+
+  * If `$color` is not a [legacy color], throw an error.
+
+  * Return the result of calling `color.channel($color, 'whiteness')`.
+  
+### `color.blackness()`
+
+* ```
+  color.blackness($color)
+  ```
+
+  * If `$color` is not a [legacy color], throw an error.
+
+  * Return the result of calling `color.channel($color, 'blackness')`.
+
+### `color.alpha()`
+
+* ```
+  color.alpha($color)
+  ```
+
+  * If `$color` is not a [legacy color], throw an error.
+
+  * Return the result of calling `color.channel($color, 'alpha')`.
+
+### `adjust-hue()`
+
+* ```
+  adjust-hue($color, $amount)
+  ```
+
+  * If `$color` is not a [legacy color], throw an error.
+
+  * Return the result of calling `color.adjust($color, $hue: $amount)`.
```

Comment:
I suppose these also likely need spaces provided. Everything that isn't just dealing with alpha.